### PR TITLE
Horovo/unet-Example: Add missing mlt.json, update readme

### PR DESCRIPTION
**Pull request by karthikvadla**
fixes #{issue_num_here_if_applicable}

If you aren't done, please add the `wip` label until you're ready for review.

If you are adding a new template or mlt command/flag ensure that is is documented in the [docs/features.md](https://github.com/IntelAI/mlt/tree/master/docs/features.md) file.

## Reviewer Checklist

 - [ ] Do you understand it?
 - [ ] Is it correct?
 - [ ] Is it safe?
 - [ ] Is it legally compliant?
 - [ ] Is it robust?
 - [ ] Is it simple?
 - [ ] Is it tested?
 - [ ] Is it documented?
 - [ ] Is the style consistent?

See [the reviewer guide](docs/reviews.md) for more detail on each check.

### CODE REVIEW

File: examples/horovod_unet/mlt_app/mlt.json [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+    "log_dir": "_multiworker",
+    "num_nodes": "2",
+    "num_workers_per_node": "1",
+    "num_inter_threads": "2",
+    "sockets_per_node": "1",
+    "gpus": "0",
+    "base_dir": "/var/datasets/unet/vck-resource-8fd827a8-809a-11e8-b982-0a580a480bd4",
+    "output_path": "/var/datasets/unet/vck-resource-8fd827a8-809a-11e8-b982-0a580a480bd4"
+  },
+  "registry": "gcr.io/constant-cubist-173123",
```
Comment (s1113950) [inline examples/horovod_unet/mlt_app/mlt.json]: Do we want to hardcode our gcr registry here?

File: examples/horovod_unet/README.md [hunk 2]
```diff
@@ -84,6 +84,20 @@ repo. Copied below list of files
     bash /src/app/exec_multiworker.sh \
     ${LOG_DIR} ${PPR} ${NUM_INTER_THREADS}"
     ```
+ 4. `NODE_SELECTOR` option - There are cases where data scientist wants to traget set of nodes to launch job.
```
Comment (s1113950) [inline examples/horovod_unet/README.md]: `target` instead of `traget` 😂

File: examples/horovod_unet/README.md [hunk 3]
```diff
@@ -84,6 +84,20 @@ repo. Copied below list of files
...
+     kubeflow openmpi component allows to set that option. We did as below.
+     In `deploy.sh` script, Look for this section.
```
Comment (s1113950) [inline examples/horovod_unet/README.md]: `Look` shouldn't be capitalized

File: examples/horovod_unet/mlt_app/mlt.json (hunk 1 above)
Comment (karthikvadla) [reply examples/horovod_unet/mlt_app/mlt.json]: oh no, i will remove, i forgot to replace.

File: examples/horovod_unet/README.md (hunk 2 above)
Comment (karthikvadla) [reply examples/horovod_unet/README.md]: 😂

**Pull request merged**